src/pages/dar_application/ResearcherInfo_new.js

```diff
@@ -196,12 +194,15 @@ export default function ResearcherInfo(props) {
             Internal Collaborators will not be required to submit an independent DAR to collaborate 
             on this project.`
           ),
-          button({
-            type: 'button', // default button element type inside a form is "submit".
-            className: 'button button-white',
-            style: { marginTop: 25 },
-            onClick: () => {}
-          }, ['Add Collaborator'])
+          h(CollaboratorList, {
+            formFieldChange,
+            collaborators: internalCollaborators,
+            collaboratorKey: 'internalCollaborators',
+            collaboratorLabel: 'Internal Collaborator',
+            showApproval: false,
+            disabled: !isEmpty(darCode),
+            deleteBoolArray: (new Array(internalCollaborators.length).fill(false)),
```

Comment:
if this always stays the same, should it not be a prop? seems like i could be computed inside of the class